src/api/Nhs.Appointments.Persistance/BookingCosmosDocumentStore.cs

```diff
@@ -73,6 +78,15 @@ public async Task<bool> UpdateStatus(string bookingReference, string status)
         return true;
     }
 
+    public async Task SetReminderSent(Booking booking)
+    {
+        var bookingDocument = _bookingStore.ConvertToDocument(booking);
+
+        var patch = PatchOperation.Replace("/reminderSent", true);
```

Comment:
You could use Set here which might be more appropriate as it does not require the property to exist